src/ui/bounding_widget_2d.h

```diff
@@ -39,6 +39,7 @@ class Bounding_Polygon_Widget {
     float split_point_size = 7.f;
     float center_point_size = 12.f;
     float selected_center_point_size = 14.f;
+    float widget_scaling_factor = 4.5f;
```

Comment:
this variable is only used on OSX, so let's wrap this in an  `ifdef` Also can we rename it to something like `macos_widget_scaling_factor` and have a comment describing why it exists (otherwise it's just a weird and confusing magic number when somebody reads the code.